### PR TITLE
Announce upgrade strategy in README.md

### PR DESCRIPTION
**Pull request by florian-richter**
Closes SAP/cloud-sdk-backlog#209

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@
 
 An SDK to reduce your development effort when building side-by-side extension applications on SAP Cloud Platform that integrate with SAP solutions and services such as SAP S/4HANA, SAP SuccessFactors, and many others.
 
+## Announcement - Upgrade Strategy
+
+Our team is working on a version 2 of the SAP Cloud SDK for JavaScript, expected to be released January 2022.
+We outlined our upgrade strategy in a post on our documentation portal: https://sap.github.io/cloud-sdk/docs/js/upgrade-strategy
```
Comment (marikaner) [inline README.md:14]:
```suggestion
We outlined our upgrade strategy [here](https://sap.github.io/cloud-sdk/docs/js/upgrade-strategy).
```

or

```suggestion
We outlined our upgrade strategy [in a post on our documentation portal](https://sap.github.io/cloud-sdk/docs/js/upgrade-strategy).
```

Or something similar?

**Review by marikaner — APPROVED** [review]
Just a minor suggestion 😊

**Comment by marikaner** [comment]
@artemkovalyov Can this be merged?

**Review by artemkovalyov — APPROVED** [review]
LGTM

**Pull request merged**